src/currency.ts

```diff
@@ -0,0 +1,14 @@
+export function SpaceCurrency(initialCurrency:number) {
```

Comment:
Name of functions shouldn't be capitalized. `generateCurrency` could be a good name for it. By the way, you can use arrow functions, look pretty good, but current one is nice too. Don't forget about return type. `function generateCurreny(n: number): string {};`.